### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by vutkin**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -2,18 +2,48 @@
...
   variables:
     EPC_REGION: AWS-EUCENTRAL
-  stage: Deployment test
+  stage: deployment test
+  script:
+    - molecule test -s cloud-aws-delegated
+  tags:
+    - delegated
+
+.EPC docker:
```
Comment (tgadiev) [inline .gitlab-ci.yml]: Why disabled test?

File: defaults/main.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 # defaults file for ansible-role-gitlab-runner
 
-gitlab_version: "9.5.0"
+gitlab_version: "10.5"
 gitlab_runner_package: "{{ gitlab_version is \
 version_compare('10.0', '>=', strict=True) | ternary('gitlab-runner','gitlab-ci-multi-runner') }}"
```
Comment (tgadiev) [inline defaults/main.yml]: `version_compare` now can be used as simple `version`

File: defaults/main.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 gitlab_host: git.epam.com
```
Comment (tgadiev) [inline defaults/main.yml:8]: Comment for line 10:
Why not break string to separate lines as yaml folded scalar accepts?

File: handlers/main.yml [hunk 4]
```diff
@@ -19,6 +19,6 @@
   become: True
```
Comment (tgadiev) [inline handlers/main.yml:19]: May be better use REST API calls by `uri` module instead of `commad`?

File: handlers/main.yml [hunk 5]
```diff
@@ -19,6 +19,6 @@
...
   listen: kick gitlab-runner
   failed_when: runner_registered.rc != 0
-  when: (not gitlab_runner_skip_registration) and gitlab_ci_token != ''
+  when: (not gitlab_runner_skip_registration) and gitlab_ci_token
```
Comment (tgadiev) [inline handlers/main.yml]: I guess it would be better looking:
```yaml
   when:
    - not gitlab_runner_skip_registration
    - gitlab_ci_token
```

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 2 above)
Comment (vutkin) [reply defaults/main.yml]: fixed

File: handlers/main.yml (hunk 4 above)
Comment (vutkin) [reply handlers/main.yml:19]: No, this is breaking change

File: handlers/main.yml (hunk 5 above)
Comment (vutkin) [reply handlers/main.yml]: Fixed

File: .gitlab-ci.yml (hunk 1 above)
Comment (vutkin) [reply .gitlab-ci.yml]: Fixed

File: defaults/main.yml (hunk 3 above)
Comment (vutkin) [reply defaults/main.yml:8]: Fixed

**Review by tgadiev — DISMISSED** [review]

**Review by tgadiev — APPROVED** [review]

**Pull request merged**